suite-common/validators/package.json

```diff
@@ -4,7 +4,7 @@
     "private": true,
     "license": "See LICENSE.md in repo root",
     "sideEffects": [
-      "src/config.ts"
+        "src/config.ts"
```

Comment:
@martykan This looks like it should not be part of this PR. It is weird since those commits are automatically generated.

Reply:
Seems like the file got in with inconsistent indentation in https://github.com/trezor/trezor-suite/commit/2adb97b8a6ec3349eab3587bc512dfe7381602a1#diff-2250600ab8a8318724b6438e69b9792df07627ab7041c18749d0acb3b4b4ecd5

CI autoformats it 

In develop it was already randomly fixed here
https://github.com/trezor/trezor-suite/commit/bb37005e6b06870ac486c68d1b761c5a2b871972#diff-2250600ab8a8318724b6438e69b9792df07627ab7041c18749d0acb3b4b4ecd5